modules/ROOT/pages/cache-scope.adoc

```diff
@@ -385,7 +385,31 @@ There are two message processors for invalidating caches:
 <ee:invalidate-key cachingStrategy-ref="InvalidatableCachingStrategy" keyGenerator-ref="MD5MuleEventKeyGenerator"/>
 ----
 
-== Example
+[IMPORTANT]
+====
+Studio does not have the Invalidate Cache component. You need to manually add the component by modifying the configuration XML, adding an `ee:invalidate-cache` element to your flow.
+====
+
+=== Example: Invalidating Cache Usage
+
+To invalidate a cache, you can create a Flow with two components:
+
+* An `http:listener` component that will be the entry point of the Flow.
+* An `ee:invalidate-cache` component that will invalidate the referenced cache.
+
+Once this Flow gets executed, it invalidates the referenced cache clearing all its stored entries.
+
+* XML Configuration file:
++
+[source,xml]
+----
+<flow name="invalidateCache">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/invalidate" doc:name="HTTP"/>
+        <ee:invalidate-cache cachingStrategy-ref="InvalidatableCachingStrategy" />
```

Comment:
you may want to also document the `keyGenerator-ref ` and `keyGenerationExpression ` attributes. Example: https://github.com/mulesoft/mule-ee/blob/mule-3.x/tests/cache/src/test/resources/config/invalidate-cache-key-config.xml

Reply:
Changes addressed in commit e8e7f09. Added more XML samples.